.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,3 @@
-Please describe your changes here.
+Please describe your changes here, explain what effect this PR will have and how this is achieved.  Refer to the # of the issue this PR addresses.  Make sure the tests run successfully using `make` target `run_tests`.
```

Comment:
`make run_tests` instead of "`make` target `run_tests`" might be less confusing.